.github/ISSUE_TEMPLATE.md

```diff
@@ -12,6 +12,11 @@
 
 **What is the [`ember-cli-rails-addon`][addon] version (from `package.json`)?**
 
+**What is your application server (e.g. `puma`, `unicorn`, `thin`, `webrick`,
+etc.)?**
+
+**Is your application server multi-threaded or multi-process?**
```

Comment:
@mike-burns good questions.

Answers to this question would provide the most useful information. The previous question is mostly a setup question, with this one as the follow up.

I split them into two because I feared that this question on its own would yield answers like "I don't know", since some developers might not put a lot of thought into choosing (or thinking about) their application server.

Would combining these two questions be an improvement?

Would omitting the list of examples be an improvment?

Could this question effectively stand on its own?